src/views/Rewards.vue

```diff
@@ -188,11 +186,11 @@ export default class Rewards extends Mixins(
           this.disconnectExternalAccountProcess();
         }
       },
-      onNetworkChange: (networkId: string) => {
-        this.setEvmNetworkType(networkId);
+      onNetworkChange: (networkHex: string) => {
+        this.connectEvmNetwork(networkHex);
```

Comment:
but rewards were only for ETH network isn't it?

Reply:
Yes, so if the user changes the metamask network to another one (the desired one is defined in `ethBridgeEvmNetwork`), we detect the change with this code and show the "change network in metamask" message.